public/components/loginForm.jsx

```diff
@@ -0,0 +1,24 @@
+import css from '../css/login.module.css';
+export default function LoginForm() {
+    return (
+        <main className={css.mainWrapper}>
+            <div className={css.blurryBackground}></div>
+            <section className={css.main}>
+                <section className={css.formWrapper}>
+                    <form className={css.form}>
+                        <h1 className={css.title}>L<span><img src="https://cdn-icons-png.flaticon.com/128/868/868596.png" /></span>gin</h1>
+                        <section className={css.inputs}>
+                            <input className={css.input} type="text" id="_username" placeholder="username..." />
+                            <input className={css.input} type="password" id="_password" placeholder="password..." />
+                            <h6 className={css.msg}></h6>
```

Comment:
not a header - it does not explain what this segment is about. change to `<p></p>`